_posts/2023-03-22-#27050.md

```diff
@@ -10,15 +10,59 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
-## Questions
-1. Did you review the PR? [Concept ACK, approach ACK, tested ACK, or NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)? What was your review approach?
--->
+- [BIP 141
+  (SegWit)](https://github.com/bitcoin/bips/blob/master/bip-0141.mediawiki)
+  introduced a new structure called a `witness` that is committed to blocks
```

Comment:
Slightly confusing if you don't mention that the witness is a new *transaction* field.
```suggestion
  introduced a new structure called a `witness` to each transaction. Transactions' witnesses are committed to blocks
```